scss/partials/service.scss

```diff
@@ -0,0 +1,214 @@
+@use 'variables.scss';
+@import url('https://fonts.googleapis.com/css2?family=Baskervville:ital@0;1&family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap');
+
+
+.services {
+    margin: 1.5rem 0;
+    background-color: variables.$primary-color;
+}
+
+.services_title {
+    text-align: center;
+    font-size: 5rem;
+    color: variables.$text-darkbrown;
+}
+
+.services_separator {
+    width: 12.5rem;
+    margin: 0 auto;
+    border-bottom: 2px solid variables.$line-pink;
+}
+
+.services_subtitle {
+    text-align: center;
+    font-size: 1.125rem;
+    color: variables.$text-black;
+    padding: 2rem;
+}
+
+.services_list {
+    background-color: variables.$background-light-pink;
+    padding: 5rem 13rem;
```

Comment:
If you use max-width, it could be better when responsive.

Reply:
As I talked with you, I changed code by removing height